recipes-support/mavlink-router/mavlink-router_0.1.bb

```diff
@@ -23,6 +23,8 @@ do_install_append () {
 	install -d ${D}${sysconfdir}/mavlink-router/config.d
 	install -m 0755 ${WORKDIR}/main.conf ${D}${sysconfdir}/mavlink-router/main.conf
 
+	install -d ${D}/home/root/logs
```

Comment:
@edersondisouza we can do this here, but mavlink-router should create the dir itself. You can use the mkdir_p function from kmod for that